lib/src/signing.rs

```diff
@@ -168,6 +170,9 @@ impl Signer {
             // Box::new(X509Backend::from_settings(settings).map_err(..)?),
         ];
 
+        #[cfg(feature = "testing")]
+        backends.push(Box::new(TestSigningBackend) as Box<dyn SigningBackend>);
```

Comment:
nit: type annotation can be omitted

and maybe it can be inlined in `vec![..]`